components/ScoreCard.vue

```diff
@@ -536,65 +536,66 @@ export default class extends Vue {
     });
   }
 
-  private lookAtManifest(): Promise<void> {
-    return new Promise(async (resolve) => {
-      try {
-        await this.getManifestInformation();
-      } catch (ex) {
-        if (this.manifest === null) {
-          this.brokenManifest = true;
-
-          this.hasHTTPS = false;
-          this.validSSL = false;
-          this.noMixedContent = false;
+  private async lookAtManifest(): Promise<void> {
+    try {
+      await this.getManifestInformation();
+    } catch (ex) {
+      if (this.manifest === null) {
+        this.brokenManifest = true;
 
-          this.securityScore = 0;
+        this.hasHTTPS = false;
+        this.validSSL = false;
+        this.noMixedContent = false;
 
-          this.$emit("securityTestDone", { score: 0 });
-        }
+        this.securityScore = 0;
 
-        this.noManifest = true;
-        resolve();
-        return;
+        this.$emit("securityTestDone", { score: 0 });
       }
 
-      if (this.manifest && this.manifest.generated === true) {
-        this.noManifest = true;
-        resolve();
-      } else {
-        this.noManifest = false;
+      this.noManifest = true;
+      return;
+    }
 
-        this.manifestScore = 15;
-        //scoring set by Jeff: 40 for manifest, 40 for sw and 20 for sc
-        if (this.manifest.display !== undefined) {
-          this.manifestScore = this.manifestScore + 5;
-        }
+    if (this.manifest && this.manifest.generated === true) {
+      this.noManifest = true;
+      return;
+    } else {
+      this.noManifest = false;
+
+      console.log(this.url);
+
+      const response = await fetch(
+        `https://pwabuilder-tests.azurewebsites.net/api/WebManifest?site=${this.url}`
```

Comment:
@jgw96 This should probably be in the environments file.

Reply:
Much agreed, that is coming (: